modules/references/client/components/create-reference/Recommend.js

```diff
@@ -17,13 +17,15 @@ export default function Recommend({
   const { t } = useTranslation('references');
 
   const recommendQuestions = {
-    hostedMe: t('Would you recommend others to stay with them?'),
-    hostedThem: t('Would you recommend others to host them?'),
-    met: t('Would you recommend others to meet them?'),
+    hostedMe: t('Did you feel comfortable being hosted by them?'),
+    hostedThem: t('Did you feel comfortable hosting them?'),
+    met: t('Did you feel comfortable meeting them?'),
   };
 
   const question = recommendQuestions[primaryInteraction];
 
+  const marginRight = { marginRight: '20px' };
```

Comment:
Let's use either styled components in JS or CSS in less files, but no inline CSS via JS.

Reply:
OK. The plan is to change upgrade react-bootstrap lib anyway and use different styling options from the new version.

Reply:
Is it resolved now?